app/controllers/api/v1/ships_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Api::V1::ShipsController < ApiController
-    def create
+  def create
```

Comment:
I'm a bit confused as to what changed here?

Reply:
Looks like I must have moved the indentation? The functionality isn't changed.